### DISCUSSION TITLE
Connection Failureconnect ECONNREFUSED 127.0.0.13306 [Apr 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1871] [Discord Threads]
> 🙋‍♂️ **smart1ne2wo**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### are you using docker?  
#  
> 👨‍🚀 **wingkwong**  
#### try to use host.docker.internal instead of localhost  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/965252437344669716)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
